### ISSUE TITLE
[feature request] Uninstall games

### ISSUE THREAD

**Issue by UnR34L** [Drop-OSS/drop-app#6]
Unless I can't see it. There's no option to uninstall games. Also, I tried manually deleting the game install folder, now it just gives an error.

- Add uninstall button
- Better handling of missing game folders, potentially mark as uninstalled if no game install directory found.

**Comment by Goosegit11** [comment]
Yeah, right now your only option is to delete `drop.db` at `~/.local/share/drop`

**Comment by DecDuck** [comment]
This feature should be ready in a few hours. 

Sorry for the mess guys! We really wanted to get something out the door by Christmas :/

**Comment by Goosegit11** [comment]
it's okay bro

**Comment by DecDuck** [comment]
Hey, I've just finished up the initial implementation. Could you please download the version for your platform from here: https://lab.deepcore.dev/drop-oss/drop-app/-/pipelines/448, and let me know if it works okay?

I've implemented both of your suggestions, so feel free to give it a test! I'll close the issue once you've confirmed.

**Comment by AdenMGB** [comment]
Can confirm it is working quite well @DecDuck
As seen in the video below it removes the game files and changes the install/play button status confirming it edits the local db

(I was running an older build and i have since noticed you fixed the colouring consistency issue)

https://github.com/user-attachments/assets/e1566b38-f145-41fd-9e98-e8dc13fb38ed
### 

All good to close it now!

**Comment by UnR34L** [comment]
Can confirm, works under normal circumstances, but I just tried installing a game that uses the "Setup executable/command". The UI is showing "Setup" but no uninstall option.

I might have used the wrong path for the setup exe. but now I'll have to reset the app to get the game uninstalled.

**Comment by DecDuck** [comment]
I realised it should also be implemented for setup too, and added it in a later commit. If you download the artifact from here: https://lab.deepcore.dev/drop-oss/drop-app/-/pipelines/477, it will include that fix.
